typings/index.d.ts

```diff
@@ -3114,6 +3114,29 @@ declare module 'discord.js' {
 
   type Status = number;
 
+  export class Sticker extends Base {
+    constructor(client: Client, data: object);
+    public asset: string;
+    public readonly createdTimestamp: number;
+    public readonly createdAt: Date;
+    public description: string;
+    public format: StickerFormatTypes;
+    public id: Snowflake;
+    public name: string;
+    public packID: Snowflake;
+    public previewAsset: string | null;
+    public tags: string | null;
+    public stickerURL(options?: StickerURLOptions): string | null;
+    public stickerPreviewURL(options?: StickerURLOptions): string | null;
+  }
+
+  interface StickerURLOptions {
+    size: ImageSize;
+    passthrough?: boolean;
+  }
+
+  type StickerFormatTypes = 'PNG' | 'APNG' | 'LOTTIE';
```

Comment:
i think this should just be `StickerFornatType`?

Reply:
djs is inconsistent with this: `MessageType`, `ChannelType`, `ActivityType`, `PartialTypes`, `MessageMentionTypes`, `WebhookTypes`